cmd/controller/controller.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/avast/retry-go"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	funk "github.com/thoas/go-funk"
```

Comment:
Do we really need this for anything else than checking if string slice contains some string? We already have that implemented internally: https://github.com/k0sproject/k0s/blob/64c1392c3914d908871a498e7a2c30b4b06a2aff/pkg/util/slice.go

Reply:
Indeed we should use our own ... I didn't see this because it's actually under `internal/` (not `pkg/`).

Reply:
Done.